module/VuFind/src/VuFind/RecordDriver/AbstractBase.php

```diff
@@ -207,17 +210,12 @@ public function getTags(
      * @param array               $tags The user-provided tags
      *
      * @return void
+     *
+     * @deprecated Use TagService::addTagsToRecord()
      */
     public function addTags($user, $tags)
     {
-        $resources = $this->getDbTable('Resource');
-        $resource = $resources->findResource(
-            $this->getUniqueId(),
-            $this->getSourceIdentifier()
-        );
-        foreach ($tags as $tag) {
-            $resource->addTag($tag, $user);
-        }
+        $this->getDbService(TagService::class)->addTagsToRecord($this, $user, $tags);
```

Comment:
Is it better practice to use the class name rather than the alias? Using the alias could we just add the `TagServiceInterface` later if needed, without having to touch the `getDbService()` calls?